app/Models/EventMeeting.php

```diff
@@ -42,7 +42,7 @@ class EventMeeting extends Model
 
     protected $guarded = [];
 
-    public function event(){
+    public function event() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->